package.json

```diff
@@ -14,9 +14,11 @@
     "react-dom": "^16.13.1",
     "react-hook-form": "^6.8.3",
     "react-input-color": "^3.0.1",
+    "react-native-anchor-point": "^1.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
     "react-select": "^3.1.0",
+    "react-spring": "^8.0.27",
```

Comment:
Please ignore both highlighted dependencies, just realised that I don't use them at all :D